log/sequencer.go

```diff
@@ -153,7 +165,9 @@ func (s Sequencer) SequenceBatch(limit int, expiryFunc CurrentRootExpiredFunc) (
 		return 0, err
 	}
 
-	leaves, err := tx.DequeueLeaves(limit)
+	// Very recent leaves inside the guard window will not be available for sequencing
+	guardCutoffTime := s.timeSource.Now().Add(-s.sequencerGuardWindow)
```

Comment:
Add a small method to get the cutoff time, so it can include `RLock()` /`RUnlock()` on the `RWMutex`